instrumentation/undertow-1.4/javaagent/build.gradle.kts

```diff
@@ -24,3 +24,11 @@ dependencies {
 tasks.withType<Test>().configureEach {
   jvmArgs("-Dotel.instrumentation.common.experimental.controller-telemetry.enabled=true")
 }
+
+// since 2.3.x, undertow is compiled by JDK 11
+val latestDepTest = findProperty("testLatestDeps") as Boolean
+if (latestDepTest) {
+  tasks.withType<JavaCompile>().configureEach {
+    options.release.set(11)
```

Comment:
Curious why you would need to add this. In CI we run latest dep tests with java 21 so even if the undertow libs require java 11 it would work.

Reply:
Though the jdk is 21, but the `--release 8` is in compile args.  So the feature on JDK9 can not be used.
this can ref `spring-boot3` module

Reply:
use
```java
if (latestDepTest) {
  otelJava {
    minJavaVersionSupported.set(JavaVersion.VERSION_11)
  }
}
```